src/client/package.json

```diff
@@ -99,12 +100,12 @@
     "uuid": "3.3.2"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^5.2.8",
-    "@storybook/addon-knobs": "^5.2.8",
-    "@storybook/addon-links": "^5.2.8",
-    "@storybook/addon-notes": "^5.2.8",
-    "@storybook/addons": "^5.2.8",
-    "@storybook/react": "^5.3.18",
+    "@storybook/addon-actions": "^6.0.28",
+    "@storybook/addon-knobs": "^6.0.28",
+    "@storybook/addon-links": "^6.0.28",
+    "@storybook/addon-notes": "^5.3.21",
```

Comment:
Is there a reason why this package's version doesn't match the others?

Reply:
there is no 6.0.28 of that package, 5.3.21 is the most recent

Reply:
just checked their github
NOTE: addon-notes has been superceded by addon-docs, which fixes numerous bugs, supports a richer feature set, and is easier to maintain. Please give it a try! https://medium.com/storybookjs/rich-docs-with-storybook-mdx-61bc145ae7bc

Reply:
Ill switch to addon-docs 6.0.28